.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: ["3.3", "3.2", "3.1", "3.0", "2.7"]
+        rest_client_version: ["1.6.0", "2.0.2", "2.1.0"]
```

Comment:
Really, 2.1.0 is already from 2019 (see https://rubygems.org/gems/rest-client). I wouldn't bother supporting older versions.